package.json

```diff
@@ -18,6 +22,16 @@
   "keywords": [
     "sd-jwt"
   ],
+  "exports": {
```

Comment:
@ragnika look at `jose` library `package.json` and make it more inline as well. Include browser version.